user-manual/access-content/search-atom.rst

```diff
@@ -736,23 +737,26 @@ with particular fields in the authority record edit template, see:
    click the |searchbutton| search button (represented by the magnifying
    glass icon to the right of the search box).
 
-.. image:: images/authority-searchbox.*
```

Comment:
If you are replacing or updating an existing image, you can literally just save a new file with an existing file name (i.e. overwrite it). Git will recognize the file has been modified and you will see it in your status checks. Right now, I'm assuming that there is still an outdated file called `authority-searchbox.png` left in the docs that we're not using?

The other option at this point, if `authority-searchbox.png` is not being used anywhere else (and `authority-search-box.png` replaces it, is to use git to delete the old file - e.g.:

```
git rm user-manual/access-content/images/authority-searchbox.png
```
That should show up in your staging area as well as something you can commit and push.

Reply:
I did delete it although with all the excitement it may or may not still be there - I'll check.

Reply:
IIRC I think you will need to remove it using git, otherwise it might lead to issues later? Anyway, no biggie either way - there are definitely some legacy images floating around in the docs I suspect.

Reply:
I think when I tried to simply overwrite an image I got an error so I changed the name slightly and saved the new images but I did delete the old images when I committed the changes and they are no longer in the folder.

Reply:
👍